core/src/test/java/org/jdbi/v3/core/argument/TestInetAddress.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.argument;
+
+import static org.junit.Assert.assertEquals;
+
+import java.net.InetAddress;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.ImmutableSet;
+
+import org.jdbi.v3.core.PGDatabaseRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class TestInetAddress {
+    @Rule
+    public PGDatabaseRule db = new PGDatabaseRule();
```

Comment:
I did some tests from this branch, and couldn't find a built in argument / mapper implementation that worked on both Postgres and H2. Postgres wants you to call `PreparedStatement.setObject(colNumber, value, Types.OTHER)` whereas H2, which doesn't have an `inet` type, can only use `VARCHAR` and wants you to call `PreparedStatement.setString`.